### ISSUE TITLE
Non-conforming drawing primitive definition `9' @ error/draw.c/RenderMVGContent/4414

### ISSUE THREAD

**Issue by labomatik** [Imagick/imagick#703]
When generating a Png picture in a docker with Swoole server Or FrankenPhp, we have this error:

```
ImagickException: non-conforming drawing primitive definition `9' @ error/draw.c/RenderMVGContent/4414
```

With the same code on the same docker using php-fpm all is correct.

Is there something that could be enabled to collect more info or details about this error?

Versions:
Php 8.1
imagick module version 3.7.0

**Comment by Danack** [comment]
> Is there something that could be enabled to collect more info or details about this error?

Not really. I mean you could try changing the [logging done internally](https://imagemagick.org/source/log.xml) by ImageMagick, but looking at the absolute eldritch horror that is the RenderMVGContent function in draw.c, there don't appear to be any logging calls internally.

I would pretty strongly recommend trying recompiling ImageMagick without OpenMP support and see if the problem 'goes away'.

About the only other thing I can think of is that changing C locale has caused an issue in ImageMagick in a similar bit of code, but that probably isn't relevant here.

**Comment by WanWizard** [comment]
Using a PHP locale that defines LC_NUMERIC as decimal comma?

Just spend 4 hours debugging the same issue, in my case `scale()` crashed with a similar message when the locale was set to "nl_NL" and `4.8765123121` was passed. No issue when changing LC_NUMERIC to "en_US" before the call.

Looks like a bug in de code, I've worked around it by switching from PNG to SVG format.
